.github/workflows/run-make-all.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           ./french_law/python/setup_env.sh
       - name: Install dependencies
         run: |
-          opam exec -- make dependencies pygments
+          OPAMCONFIRMLEVEL=unsafe-yes opam exec -- make dependencies pygments
```

Comment:
Otherwise, opam won't accept to use `sudo` and install system dependencies without user confirmation. Note that if we migrate to a Docker image they generally already export this to the environment.